test/manual/analytics-batching.amp.html

```diff
@@ -0,0 +1,74 @@
+</p>
+<p>
+Integer dapibus egestas arcu. Nunc vitae velit congue, placerat augue quis, suscipit nisi. Donec suscipit imperdiet turpis pharetra feugiat. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Phasellus aliquam eleifend dolor, at lacinia orci semper vel. Nunc semper sem vel tincidunt posuere. Nunc lobortis velit vitae condimentum mollis. Morbi eu ullamcorper mauris. Pellentesque ac eros maximus, pulvinar sapien vitae, semper nisi. Curabitur imperdiet non mauris vitae sollicitudin.
+</p>
+<p>
+Nam posuere velit euismod risus pulvinar, in sollicitudin sapien consectetur. Vestibulum nec ex odio. Quisque at elit nec nunc ultricies lacinia nec non lorem. Maecenas porttitor consequat mauris, vitae porttitor ligula pellentesque ut. Pellentesque rhoncus diam vel lacus lobortis imperdiet. Sed maximus dictum hendrerit. Vivamus ornare, purus in laoreet sagittis, est ante pretium mauris, vel vulputate arcu erat eget mauris. Suspendisse eu lorem metus. Aliquam tempus aliquet urna, vitae mollis lacus pretium vitae. Etiam semper gravida commodo. Maecenas at pulvinar quam. Nullam dolor ipsum, ornare a sollicitudin et, sodales porttitor neque.
+</p>
+<p>
+Integer in felis at lacus mattis facilisis. Curabitur tincidunt, felis porttitor mollis finibus, tortor elit elementum dolor, vel vulputate lorem dui id ante. Vivamus in velit at lectus blandit gravida vitae quis arcu. Nam et magna magna. Fusce condimentum diam lacus, ac ullamcorper purus malesuada eu. Mauris ullamcorper elit et venenatis faucibus. Nullam lobortis molestie purus quis pellentesque. Sed at libero id nisi rhoncus tincidunt. Praesent vestibulum vehicula tristique. Etiam rutrum, nunc id porta interdum, nulla nisi molestie leo, at fermentum justo dolor at lorem. Duis in egestas sapien.
+</p>
+<amp-analytics type="_ping_">
+  <script type="application/json">
+  {
+    "requests-v2": {
+      "old": "https://fake.com/analytics?timer",
+      "batch": {
+        "baseUrl": "https://fake.com/batch-analytics?",
+        "maxDelay": 5
+      }
+    },
+    "triggers": {
+      "timer": {
+        "on": "timer",
+        "request": "old",
+        "timerSpec": {
+          "interval": 2,
+          "maxTimerLength": 10
+        }
+      },
+      "batch-timer": {
+        "on": "timer",
+        "request": "batch",
+        "timerSpec": {
+          "interval": 2,
+          "maxTimerLength": 10
+        },
+        "batchSegment": "&timer",
+        "extraUrlParams": {
+          "extra": "34",
+          "extra2": "${requestCount}"
+        }
+      }
+    }
+  }
+  </script>
+</amp-analytics>
```

Comment:
What is the purpose of this example? To be loaded and examined locally? As is, the main way to track what's happening would be opening the console. Perhaps force transport to image so it's easier to track in the network tab.

Reply:
good idea. changed transport method to image.